Api/Data/ProductCacheInterface.php

```diff
@@ -46,9 +46,7 @@ interface ProductCacheInterface
     const PRODUCT_ID = 'product_id';
     const CREATED_AT = 'created_at';
     const UPDATED_AT = 'updated_at';
-    const IN_SYNC = 'in_sync';
```

Comment:
Needs to be removed from here as well: https://github.com/Nosto/nosto-magento2/pull/663/files#diff-5e2225a16361e4d182e589fecf957e82R168